frameworks/cassandra/tests/test_auth.py

```diff
@@ -6,11 +6,18 @@
 import sdk_cmd
 import sdk_install
 import sdk_tasks
+import os
 from tests import config
 
 log = logging.getLogger(__name__)
 
 
+no_auth_test_for_open_dcos = pytest.mark.skipif(
```

Comment:
This looks near identical to the `sdk_utils.dcos_ee_only` method [here](https://github.com/mesosphere/dcos-commons/blob/DCOS-53489-cassandra-test-auth-failing-on-DC/OS-Open/testing/sdk_utils.py#L198). No need to reinvent, can you import and use the method instead?

Reply:
Correct @samvantran , 
this is what I did previously,import sdk_utils, and marked @sdk_utils.dcos_ee_only  # at top of test.
but it was not working, so I had to use this way.